testsuite/tools/expect_test.ml

```diff
@@ -181,8 +181,8 @@ let capture_everything buf ppf ~f =
                      ~f:(fun () -> Compiler_messages.capture ppf ~f)
 
 let exec_phrase ppf phrase =
-  if !Clflags.dump_parsetree then Printast. top_phrase ppf phrase;
-  if !Clflags.dump_source    then Pprintast.top_phrase ppf phrase;
+  if !Clflags.dump_parsetree then Printast.top_phrase ppf phrase;
+  if !Clflags.dump_source then Pprintast.top_phrase ppf phrase;
```

Comment:
That diff is a bit gratuitous.

Reply:
Oh yeah, that's probably leftover from previous changes, I'll remove it while I squash all the commits.